server/rest/src/test/java/org/infinispan/rest/resources/CacheManagerResourceTest.java

```diff
@@ -213,6 +212,7 @@ public void testInfo() throws Exception {
       assertFalse(cmInfo.get("version").asText().isEmpty());
       assertEquals(2, cmInfo.get("cluster_members").size());
       assertEquals(2, cmInfo.get("cluster_members_physical_addresses").size());
+      assertTrue(cmInfo.get("local_site").isNull());
```

Comment:
I'm not sure how useful this test is :)

Reply:
Test that the property is there with than name. Rest api contract

Reply:
well, it doesn't test if it returns the correct name... I'll close my eyes :)

Reply:
is there an easy way to configure the local site name for the test?

Reply:
use the method that accepts the `TransportFlags` when instantiating the `CacheManager` and  use `TransportFlags.minimalXsiteFlags()`
The local site name will be `LON-1`